draft-ietf-httpbis-digest-headers.md

```diff
@@ -592,9 +593,15 @@ Digest: sha-256=X48E9qOokqqrvdts8nOJRJN3OWDUoyWxBf7kbu9DBPE=
 "hello"
 ~~~
 
-### Digest in both Request and Response. Returned value depends on representation metadata
+### Digest in both Request and Response. Field-values depends on representation metadata
+
+The request contains a `Digest` header
+calculated on the enclosed representation.
 
-Digest can be used in requests too. Returned value depends on the representation metadata header fields.
+The `Digest` field-value in the response is different
+from the one of the request
+because we used `Accept-Encoding` to selected another
+representation respect to the sent one.
```

Comment:
Then reword this as something like:

"The response includes a `Content-Encoding:br` that indicates the selected representation is brotli encoded. The `Digest` field-value is therefore different compared to the request.